sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/UnnamedSessionManager.java

```diff
@@ -160,18 +161,18 @@ String getLinkName(String sessionId) {
      * @return The next expiration time for the session lock.
      * @throws IllegalStateException if the receiver is a non-session receiver.
      */
-    Mono<Instant> renewSessionLock(String sessionId) {
+    Mono<OffsetDateTime> renewSessionLock(String sessionId) {
         return validateParameter(sessionId, "sessionId", "renewSessionLock").then(
             getManagementNode().flatMap(channel -> {
                 final UnnamedSessionReceiver receiver = sessionReceivers.get(sessionId);
                 final String associatedLinkName = receiver != null ? receiver.getLinkName() : null;
 
                 return channel.renewSessionLock(sessionId, associatedLinkName).handle((instant, sink) -> {
                     if (receiver != null) {
-                        receiver.setSessionLockedUntil(instant);
+                        receiver.setSessionLockedUntil(instant.atOffset(ZoneOffset.UTC));
```

Comment:
I thought we made managementNode.renewSessionLock return OffsetDateTime. You had changed one operation in there.

Reply:
Will refactor it with issue #14691